src/include/oslogin_sshca.h

```diff
@@ -23,10 +23,6 @@
 #include <sys/syslog.h>
 #include <sys/types.h>
 
-#ifdef __cplusplus
```

Comment:
why did this exist before? was there a distro that depended on it?

Reply:
basically it was a: "I wrote it in C" rather then C++, no actual reason.